Databricks/Alluxio-2.9.0-without-CVE/build-from-src-and-fix-CVE.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+# clone, apply patches and compile
```

Comment:
add more description as to what this is doing and why.

Reply:
Done